apps/wfo-ui-surf/package.json

```diff
@@ -3,7 +3,7 @@
     "version": "1.0.0",
     "private": true,
     "scripts": {
-        "dev": "next dev",
+        "dev": "next dev -p 3001",
```

Comment:
This could prove problematic when combined with auth.

The backend allow "*" origin; but I suspect that keycloak will be more strict

Reply:
Why fix the port again?

Reply:
In case you run both apps at the same time with `dev:all` it is a race condition which app runs on port 3000 vs 3001. Fixing the port numbers like this give you more certainty what to expect on each port.